app/views/articles/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with model: @article, class: "form-control" do |form| %>
```

Comment:
If you're using the instance variable in the partial you don't need to pass it as an argument when you render.

If you use this 
```
<%= render "form", article: @article %>
```
you don't need the `@` here
```
<%= form_with model: @article, class: "form-control" do |form| %>
```

Reply:
Ah gotcha! Thanks for pointing that out!